spring-kafka/src/main/java/org/springframework/kafka/annotation/EnableKStreams.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2016-2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.kafka.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.springframework.context.annotation.Import;
+import org.springframework.kafka.core.KStreamBuilderFactoryBean;
+
+/**
+ * Enable Default Kafka Streams components. To be used on
+ * {@link org.springframework.context.annotation.Configuration Configuration} classes as
+ * follows:
+ *
+ * <pre class="code">
+ * &#064;Configuration
+ * &#064;EnableKStreams
+ * public class AppConfig {
+ *
+ * 	&#064;Bean(name = KStreamDefaultConfiguration.DEFAULT_STREAMS_CONFIG_BEAN_NAME)
+ *  public StreamsConfig kStreamsConfigs() {
+ *     ...
+ *  }
+ * 	// other &#064;Bean definitions
+ * }
+ * </pre>
+ *
+ * That {@link KStreamDefaultConfiguration#DEFAULT_STREAMS_CONFIG_BEAN_NAME} is required
+ * to declare {@link KStreamBuilderFactoryBean} with the
+ * {@link KStreamDefaultConfiguration#DEFAULT_KSTREAM_BUILDER_BEAN_NAME}.
+ * <p>
+ * Also to enable KStreams feature you should be sure that {@code kafka-streams} jar is
+ * on classpath.
+ *
+ * @author Artem Bilan
+ *
+ * @since 2.0
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@Import(KStreamDefaultConfiguration.class)
+public @interface EnableKStreams {
```

Comment:
IMO the name here should be either `@EnableKafkaStreams` or `@EnableKStream`. Maybe the former would be more correct - given that `KStream` is the API type and 'Kafka Streams' is the official name of the feature.